spring-boot-modules/spring-boot-crud/src/main/java/com/baeldung/entity_dto_differences/entity/Book.java

```diff
@@ -0,0 +1,38 @@
+package com.baeldung.entity_dto_differences.entity;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "books")
+public class Book {
+
+    @Id
+    private String name;
+    private String author;
+
+    public Book() {
```

Comment:
Since you left a comment about the reason for the default constructor in the other classes, I think it makes sense to do the same here too.